at_app_flutter/pubspec.lock

```diff
@@ -0,0 +1,565 @@
+# Generated by pub
```

Comment:
I'm not sure that this file should be committed:

Per https://dart.dev/guides/libraries/private-files#pubspeclock

> For library packages, don’t commit the pubspec.lock file. Regenerating the pubspec.lock file lets you test your package against the latest compatible versions of its dependencies.

Reply:
Removed in the latest commit.